package/carbonio-mta-sidecar.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Carbonio mta sidecar proxy
+Documentation=https://docs.zextras.com/zextras-suite-documentation/latest/
```

Comment:
Consider using this documentation URL like we do everywhere: 
https://docs.zextras.com/

If required, template repo should also be updated.